src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -400,22 +426,85 @@ long TriangleCountExecutor::getTriangleCount(int graphId, std::string host, int
             triangleCount_logger.log("Got response : |" + response + "|", "info");
             response = utils.trim_copy(response, " \f\n\r\t\v");
             triangleCount = atol(response.c_str());
-            workerResponded = true;
         }
 
         if (isCompositeAggregation) {
+
+            triangleCount_logger.log("###COMPOSITE### Started Compsite aggregation ", "info");
             static std::vector<std::vector<string>>::iterator combinationsIterator;
 
             for (int combinationIndex = 0; combinationIndex < fileCombinations.size(); ++combinationIndex) {
                 std::vector<string> fileList = fileCombinations.at(combinationIndex);
                 std::vector<string>::iterator fileListIterator;
+                std::vector<string>::iterator listIterator;
                 std::set<string> partitionIdSet;
+                std::set<string> partitionSet;
+                std::map<int,int> tempWeightMap;
+                std::set<string>::iterator partitionSetIterator;
                 std::set<string> transferRequireFiles;
                 std::string combinationKey = "";
                 std::string availableFiles = "";
                 std::string transferredFiles = "";
                 bool isAggregateValid = false;
 
+                for (listIterator = fileList.begin(); listIterator != fileList.end(); ++listIterator) {
+                    std::string fileName = *listIterator;
+
+                    size_t lastindex = fileName.find_last_of(".");
+                    string rawFileName = fileName.substr(0, lastindex);
+
+                    std::vector<std::string> fileNameParts = utils.split(rawFileName,'_');
+
+                    for (int index = 2; index < fileNameParts.size(); ++index) {
```

Comment:
Why stat from 2 here?

Reply:
Once the rawFileName splitted by underscore the partition data always contains after index 2. We only need partition information for this process. Therefore we are starting the loop with index 2.

Reply:
A comment was added to the source code explaining the reason for starting with 2

Reply:
Ok, thanks for the clarification. I still do not see the comment though.

Reply:
If you go to the latest file the changes are available

Reply:
Ok